packages/client/src/clients/feed/feed.ts

```diff
@@ -583,12 +586,17 @@ class Feed {
   private async makeStatusUpdate(
     itemOrItems: FeedItemOrItems,
     type: MessageEngagementStatus | "unread" | "unseen" | "unarchived",
+    metadata?: Record<string, string>,
```

Comment:
Maybe a nit but should we consider changing this this third function param to an object that can contain a meta data var? Starts to get smelly after 2 function parameters. Same for the markAsInteracted.